### PR TITLE
Dockerfile for fastText 

### PR DESCRIPTION
**Pull request by xeb**
I thought I'd share the Dockerfile I've been using during training and hopefully serving one day soon.  I'd suggest publishing in DockerHub.

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by alexbeletsky** [comment]
Good stuff, working in container instead of machine makes a lot of sense. Does this image include all required python tools as well?

File: README.md [hunk 1]
```diff
@@ -28,6 +28,18 @@ $ make
 This will produce object files for all the classes as well as the main binary `fasttext`.
 If you do not plan on using the default system-wide compiler, update the two macros defined at the beginning of the Makefile (CC and INCLUDES).
 
+## Docker Build of fastText
+
+In order to build `fastText` with DOcker, use the following:
```
Comment (pyk) [inline README.md]: `DOcker` should be `Docker`

**Comment by pyk** [comment]
@alexbeletsky The current `Dockerfile` doesn't install the `python 2.6

**Comment by piotr-bojanowski** [comment]
Hi,

We find the idea of writing a docker file very interesting! However, since it is not core to the library, could you put it in another repository that we mention in the readme?

Best Regards

Piotr

**Comment by xeb** [comment]
Separate repository for the Dockerfile created!  Apologies for the delay & reopen.  Not sure where in the README.md you'd prefer to have the link, so I put it near the Building header.

The example Docker repository includes: 
- Ubuntu 14.04 base image (needed certain glibc headers for scipy install)
- Python dependencies
- An example script of the classification-example.sh using only Docker containers
- Makefile to build the container, run it (with bash) and run the example.

Happy to include anything else others find value.

[https://github.com/xeb/fastText-docker](https://github.com/xeb/fastText-docker)

File: README.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ $ make
 This will produce object files for all the classes as well as the main binary `fasttext`.
 If you do not plan on using the default system-wide compiler, update the two macros defined at the beginning of the Makefile (CC and INCLUDES).
 
+### Building with Docker
+
+To build with Docker, see [fastText-Docker](https://github.com/xeb/fastText-docker) repository.
```
Comment (piotr-bojanowski) [inline README.md]: I would rather be less directive and simply advertise that a docker build file is available at.
Something like: "If you want to build with docker, please check ...".
Thanks!

**Comment by piotr-bojanowski** [comment]
I pulled your readme and corrected the wording. Thanks!

**Pull request merged**